packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -50,6 +50,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)captureToFile:(FLTThreadSafeFlutterResult *)result API_AVAILABLE(ios(10));
 - (void)close;
 - (void)startVideoRecordingWithResult:(FLTThreadSafeFlutterResult *)result;
+- (void)startVideoRecordingWithResult:(FLTThreadSafeFlutterResult *)result
+                       streamCallback:(nullable NSObject<FlutterBinaryMessenger> *)messenger;
```

Comment:
maybe `messengerForStreaming`? this should not be named `streamCallback` since it's not a callback block.

Reply:
Will do.